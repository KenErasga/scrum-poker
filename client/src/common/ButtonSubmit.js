import React from 'react';

import { Button } from '@material-ui/core';

const ButtonSubmit = ({ description }) => {
    return (
            <Button
                type='submit' fullWidth
                variant="contained"
                color="primary"
                aria-label={description}
                >
                    {description}
            </Button>
    );
}

export default ButtonSubmit;