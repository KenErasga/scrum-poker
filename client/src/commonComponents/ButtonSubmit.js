import React from 'react';

import { Button } from '@material-ui/core';

const ButtonSubmit = ({ description, sessionName, name }) => {
    return (
            <Button
                type='submit' fullWidth
                variant="contained"
                color="primary"
                >
                    {description}
            </Button>
    );
}

export default ButtonSubmit;