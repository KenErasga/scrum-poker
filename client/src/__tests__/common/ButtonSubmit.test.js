import React from 'react';
import renderer from 'react-test-renderer';
import ButtonSubmit from '../../common/ButtonSubmit';

it('ButtonSubmit renders correctly', () => {
  const tree = renderer
    .create(<ButtonSubmit description='test-description'></ButtonSubmit>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});