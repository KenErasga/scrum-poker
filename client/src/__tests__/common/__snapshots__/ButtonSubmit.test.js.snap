// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonSubmit renders correctly 1`] = `
<button
  aria-label="test-description"
  className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-fullWidth"
  disabled={false}
  onBlur={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex={0}
  type="submit"
>
  <span
    className="MuiButton-label"
  >
    test-description
  </span>
</button>
`;
